src/brogue/RogueMain.c

```diff
@@ -154,6 +187,11 @@ void initializeRogue(uint64_t seed) {
     wizard = rogue.wizard;
     displayStealthRangeMode = rogue.displayStealthRangeMode;
     trueColorMode = rogue.trueColorMode;
+
+    if (rogue.meteredItems != NULL) {
```

Comment:
The number of possible metered items in the game now depends on variant so needs to be dynamic